values-sk/strings.xml

```diff
@@ -145,9 +145,11 @@
     <string name="draft_gillender00_text">Budova Gillender z New Yorku. Ďakujeme za 250 000 stiahnutí :D</string>
     <string name="topic_thanks">Ďakujeme za úžasnú podporu!</string>
     <string name="topic_cake">Torta je falošná!</string>
-    <string name="topic_panic">Nepanikárte</string>
+    <string name="topic_panic">Nepanikárte!!!</string>
     <string name="draft_bigschool00_title">Stredná škola</string>
-    <string name="draft_bigschool00_text">Učia sa tu žiaci po absolvovaní základnej školy.</string>
+    <string name="draft_bigschool00_text">Učia sa tu žiaci po absolvovaní základnej školy. (
+\n
+\n(pozn. V našej hra funguje rovnako ako základná škola)</string>
```

Comment:
I am not sure this clarification is needed.

Reply:
I just wanted to text look better. If it's problem, i can reedit it.

Reply:
If you feel it's better this way, then I see no problem.